fontforge/parsepdf.c

```diff
@@ -1942,6 +1942,8 @@ static SplineFont *pdf_loadtype3(struct pdfcontext *pc) {
 return( sf );
 
   fail:
+    if ( name!=NULL )
```

Comment:
You can remove the `if` check, a free of NULL is fine

Reply:
Gotcha. Done. I appreciate you taking the time to guide me through this - I know teaching the fundamentals of C wasn't on your todo list for today!